### ISSUE TITLE
Personalized preferences

### ISSUE THREAD

**Issue by rlogiacco** [rlogiacco/Natural#15]
The plugin is showing two preference pages that are not needed: compiler and refactor.

They should both be removed, unless refactoring can be implemented in a nice way.
##

**Comment by drkstr101** [comment]
Flagging for review in [next](https://github.com/rlogiacco/Natural/pull/73).

**Comment by drkstr101** [comment]
Step refactoring across feature files may be a handy feature. This was always a pain when there are a lot of compostable steps used everywhere.

**Comment by rlogiacco** [comment]
Yes, but can you imagine the pain in doing this with outlines and examples?
